nameserver/zone.go

```diff
@@ -18,6 +18,7 @@ var rdnsDomainLen = len(RDNSDomain) + 1
 
 type Zone interface {
 	AddRecord(ident string, name string, ip net.IP) error
+	DeleteRecords(ident *string, ip *net.IP, name *string) (count int)
```

Comment:
I'd argue that the error return on the other methods is wrong too. The only method were it makes sense would be one that takes all three params, non-wildcarded. i.e. the proper inverse of `AddRecord`.